data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaConnection.java

```diff
@@ -0,0 +1,12 @@
+package org.opensearch.dataprepper.plugins.kafka.configuration;
+
+import java.util.Collection;
+import java.util.List;
+
+public interface KafkaConnection {
```

Comment:
Suggest using a better name like `KafkaBaseConfig` because this is not really a KafkaConnection